.drone.star

```diff
@@ -1,5 +1,14 @@
 def main(ctx):
     versions = [
+        {
+            "value": "10.12.1-rc.2",
+            "qa": "https://download.owncloud.com/server/testing/owncloud-complete-20230404-qa.tar.bz2",
```

Comment:
Please remove the QA tarball. Daily QA bundle is used by default.

Reply:
removed.